src/SketchRNN/index.js

```diff
@@ -3,23 +3,23 @@
 // This software is released under the MIT License.
 // https://opensource.org/licenses/MIT
 
-/* eslint prefer-destructuring: ['error', {AssignmentExpression: {array: false}}] */
-/* eslint no-await-in-loop: 'off' */
 /*
 SketchRNN
 */
 
 import * as ms from '@magenta/sketch';
 import callCallback from '../utils/callcallback';
-import modelPaths from './models';
+import handleArguments from "../utils/handleArguments";
 import modelLoader from '../utils/modelLoader';
+import modelPaths from './models';
 
 // const PATH_START_LARGE = 'https://storage.googleapis.com/quickdraw-models/sketchRNN/large_models/';
 // const PATH_START_SMALL = 'https://storage.googleapis.com/quickdraw-models/sketchRNN/models/';
 // const PATH_END = '.gen.json';
 
 
 const DEFAULTS = {
+  // TODO: this doesn't look right... -Linda
```

Comment:
nonblocking comment:
* Oh weird! Hmm seems like we might add this to another follow up to resolve. 🙈